modules/ossm-uninstalling-cert-manager.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-uninstalling-cert-manager_{context}"]
+= Uninstalling Service Mesh with the cert-manager Operator using the Istio-CSR agent
+
+You can uninstall the cert-manager with the {SMProduct} by using the following procedure. Before removing the following resources, verify that no {SMProductName} or {istio} components reference the `Istio-CSR` service or the certificates it issued. Removing these resources while they are still in use may disrupt mesh functionality.
+
+.Procedure
+
+. Remove the `IstioCSR` custom resource by running the following command:
++
+[source, terminal]
+----
+$ oc -n <istio-csr_project_name> delete istiocsrs.operator.openshift.io default
+----
+
+. Remove the related resources:
+
+.. List the cluster scoped-resources by running the following command:
++
+[source, terminal]
+----
+$ oc get clusterrolebindings,clusterroles -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr"
+----
++
+Save the names of the listed resources for later reference.
+
+.. List the resources in `Istio-csr` agent deployed namespace by running the following command:
++
+[source, terminal]
+----
+$ oc get certificate,deployments,services,serviceaccounts -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr" -n <istio_csr_project_name>
+----
++
+Save the names of the listed resources for later reference.
+
+.. List the resources in {SMProductName} or {istio} deployed namespaces by running the following command:
++
+[source, terminal]
+----
+$ oc get roles,rolebindings -l "app=cert-manager-istio-csr,app.kubernetes.io/name=cert-manager-istio-csr" -n <istio_csr_project_name>
```

Comment:
Consider reformatting this long command so it is easier to read: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#backslash-for-long-lines-of-code